src/Sylius/Behat/Page/SymfonyPage.php

```diff
@@ -77,6 +77,25 @@ protected function waitForModalToAppear(NodeElement $modalContainer)
     }
 
     /**
+     * @param NodeElement $collection
+     *
+     * @return NodeElement|bool
+     */
+    protected function waitForCollectionItemToAppear(NodeElement $collection)
+    {
+        $itemCount = count($collection->findAll('css', 'div[data-form-collection=item]'));
+
+        $this->getDocument()->waitFor(1, function () use ($itemCount, $collection) {
+            $items = $collection->findAll('css', 'div[data-form-collection=item]');
+            if (count($items) === $itemCount + 1) {
+                return end($items);
+            }
+
+            return false;
```

Comment:
It's strange when method returns ``NodeElement`` or ``false`` :| maybe it should throw exception in case if condition is not fulfilled?